### PR TITLE
RTMPConnection: expose peer address

### PR DESCRIPTION
**Pull request by mildsunrise**
when reviewing code I found a TODO for this, thought it was quick enough to implement

### CODE REVIEW

File: include/rtmp/rtmpapplication.h [hunk 1]
```diff
@@ -4,11 +4,17 @@
...
 #include <string>
 #include <functional>
+#include <sys/socket.h>
 
 class RTMPApplication
 {
 public:
-	virtual RTMPNetConnection::shared Connect(const std::wstring& appName,RTMPNetConnection::Listener *listener,std::function<void(bool)> accept) = 0;
+	virtual RTMPNetConnection::shared Connect(
+		struct sockaddr_in peername,
```
Comment (murillo128) [inline include/rtmp/rtmpapplication.h:13]: do we need the `struct`? pass by const & better

File: src/rtmp/rtmpconnection.cpp [hunk 2]
```diff
@@ -945,6 +945,16 @@ void RTMPConnection::ProcessCommandMessage(DWORD streamId,RTMPCommandMessage* cm
 			//Send error
 			return SendCommandError(streamId,transId);
 
+		//Get client address
+		struct sockaddr_in peername;
+		socklen_t peername_len = sizeof(peername);
```
Comment (murillo128) [inline src/rtmp/rtmpconnection.cpp:950]: use cammel case

**Review by murillo128 — COMMENTED** [review]
good one, just minor nits

**Comment by mildsunrise** [comment]
both things fixed, merging

**Pull request merged**